diagnosis/getkuberneteslogs.sh

```diff
@@ -298,6 +315,11 @@ then
     done
 fi
 
+if [ -n "$API_MODEL" ]
+then
+    cp $API_MODEL $LOGFILEFOLDER
```

Comment:
Because this file is already on the user file system, it only makes sense to move it to SA_DIR